inc/woocommerce/storefront-woocommerce-template-hooks.php

```diff
@@ -64,6 +64,13 @@
 add_action( 'storefront_header', 'storefront_product_search', 40 );
 add_action( 'storefront_header', 'storefront_header_cart',    60 );
 
+/**
+ * Structured Data
+ *
+ * @see storefront_woocommerce_init_structured_data()
+ */
+add_action( 'woocommerce_before_shop_loop_item', 'storefornt_woocommerce_init_structured_data' );
```

Comment:
I was wondering how it managed to work with your pointed typo error.
Surprisingly, I made the same here without copy pasting the function name !